iexec-sms-library/src/main/java/com/iexec/sms/api/config/TeeServicesProperties.java

```diff
@@ -32,6 +32,15 @@
 // TODO upgrade to sealed class in Java 17
 public abstract class TeeServicesProperties {
     private final TeeFramework teeFramework;
+    private final String teeFrameworkVersion;
     private final TeeAppProperties preComputeProperties;
     private final TeeAppProperties postComputeProperties;
+
+    @Deprecated(forRemoval = true)
+    protected TeeServicesProperties(TeeFramework teeFramework, TeeAppProperties preComputeProperties, TeeAppProperties postComputeProperties) {
+        this.teeFramework = teeFramework;
+        this.teeFrameworkVersion = "";
+        this.preComputeProperties = preComputeProperties;
+        this.postComputeProperties = postComputeProperties;
```

Comment:
As we have the `@AllArgsConstructor` annotation, this can be replaced with
```suggestion
        this(teeFramework, "", preComputeProperties, postComputeProperties);
```
This will help for code coverage

Reply:
Resolved in [75e68d2](https://github.com/iExecBlockchainComputing/iexec-sms/pull/289/commits/75e68d2b2c9b33c73e83b23f2d933a4758ff25e0)